infra/helm/meshdb/templates/meshweb.yaml

```diff
@@ -47,18 +47,38 @@ spec:
               mountPath: /opt/meshdb/static
           {{ if eq .Values.meshweb.liveness_probe "true" }}
           livenessProbe:
-           exec:
-             command:
-               - curl
-               - http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1
-           periodSeconds: 3
-           initialDelaySeconds: 4
-           timeoutSeconds: 3
-           {{ end }}
-          livenessProbe:
-            {{- toYaml .Values.livenessProbe | nindent 12 }}
+            exec:
+              command:
+                - bash
+                - -c
+                - 'curl http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1/ -H "Host: db.nycmesh.net" -s | grep meshin'
+            periodSeconds: 10
+            initialDelaySeconds: 30
+            timeoutSeconds: 3
+          {{ end }}
+          {{ if eq .Values.meshweb.readiness_probe "true" }} 
           readinessProbe:
-            {{- toYaml .Values.readinessProbe | nindent 12 }}
+            exec:
+              command:
+                - bash
+                - -c
+                - 'curl http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1/ -H "Host: db.nycmesh.net" -s | grep meshin'
+            periodSeconds: 3
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+          {{ end }}
+          {{ if eq .Values.meshweb.startup_probe "true" }} 
+          startupProbe:
+            exec:
+              command:
+                - bash
+                - -c
+                - 'curl http://127.0.0.1:{{ .Values.meshweb.port }}/api/v1/ -H "Host: db.nycmesh.net" -s | grep meshin'
+            periodSeconds: 3
+            initialDelaySeconds: 20
+            timeoutSeconds: 3
+            failureThreshold: 20
+          {{ end }}
```

Comment:
I don't think it's worth having redundant probes. We should consider what kind of probes to use in place of these. Maybe we should just check if we get anything back from nginx at all as the liveness probe, then make the readiness probe check `we're meshin'`?

Not sure about startupProbe

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
Actually, I think I might be wrong to suggest that.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

> A common pattern for liveness probes is to use the same low-cost HTTP endpoint as for readiness probes, but with a higher failureThreshold. This ensures that the pod is observed as not-ready for some period of time before it is hard killed.